### PR TITLE
JSON Validator: Design Tweaks

### LINKED ISSUE #3474: Json Validator: Implement design tweaks
There are some minor design changes that should be made:

### CSS
- `<textarea>` should have `monospace` font family -- remove quotes around `'monospace'` in CSS
- Only enable validate button once the `<textarea>` has content
- Add `whole` class to `<textarea>` and `<pre>`
- Use proper CSS comment syntax `/* */` for `overflow: auto`

Should look like:

```css
.zci--json_validator .json_validator--input {
    margin-bottom: 1.5em;
    font-family: monospace;
    overflow: auto;  /* Hide default scrollbars on IE */
}

.zci--json_validator button {
    cursor: default;
}

.zci--json_validator .json_validator--result__wrap {
    margin-top: 1em;
}

/* Mobile */
.is-mobile .zci--json_validator button {
    text-align: center;
    padding-left: 0;
    padding-right: 0;
}

.is-mobile .zci--json_validator textarea {
    resize: vertical;
}
```

### HTML
- Remove "gw" wraps from `<textarea>`, `<btn>` and `<pre>` --- these are only necessary when we have multiple columns in a `<gw>` element
- set `tx--12` font size for result
- add `Results` heading to output

Should look like: 

```html
<textarea name="json_validator--input" class="json_validator--input whole tx--12 frm__text" rows="{{cols}}" cols="60" placeholder="Paste code here..."></textarea>

<button class="btn btn--skeleton whole--screen-s" disabled>Loading..</button>

<div class='json_validator--result__wrap is-hidden'>
    <h6 class="text--secondary">Results:</h6>
    <pre class="json_validator--result whole tx--13"></pre>
</div>
```

### JS
- Do not remove `c-base` class from container -- this is not necessary
- update to match adjusted HTML
- change `cols` to `rows` -- that was a typo I made on the last PR...

### Result:

json_validator_at_duckduckgo

json_validator_at_duckduckgo

The current design is OK, but I'm sure we can make this better for V2.

------
IA Page:  http://duck.co/ia/view/json_validator 
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @sahildua2305

### PR DESCRIPTION
**Pull request by sahildua2305**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] New Instant Answer
    - [ ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [x] Improvement
    - [x] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

Provide an overview of the changes this pull request introduces.

###### Which issues (if any) does this fix?

Fixes #3474 

###### People to notify (@mention interested parties)
@moollaza 

---

Instant Answer Page: https://duck.co/ia/view/json_validator

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/jsonvalidator/content.handlebars [hunk 1]
```diff
@@ -1,17 +1,9 @@
...
-</div>
+<textarea name="json_validator--input" class="json_validator--input whole tx--12 frm__text" rows="{{rows}}" cols="60" placeholder="Paste code here..."></textarea>
 
-<div class="gw">
-    <div class="g json_validator--action_box">
-        <button class="btn btn--skeleton" disabled>Loading..</button>
-    </div>
-</div>
+<button class="btn btn--skeleton whole--screen-s json_validator--validate_button" disabled>Loading..</button>
+<button class="btn btn--critical whole--screen-s json_validator--clear_button">Clear</button>
```
Comment (moollaza) [inline share/goodie/jsonvalidator/content.handlebars]: Let's us `button--secondary` -- Critical makes this appear as the most important button on the page, when it should really be something subtle.

**Comment by moollaza** [comment]
@sahildua2305 aside from the clear button, this LGTM :+1:

**Comment by vikashvverma** [comment]
@sahildua2305 Validate button should remain disabled when there is no input

File: share/goodie/jsonvalidator/content.handlebars (hunk 1 above)
Comment (sahildua2305) [inline share/goodie/jsonvalidator/content.handlebars]: Makes sense! I'll change this asap.

**Comment by moollaza** [comment]
@sahildua2305 one more minor fix, clicking "Clear" should empty the results div and disable the Validate button

**Comment by sahildua2305** [comment]
@moollaza We are good to go ahead with this. :+1:

**Comment by moollaza** [comment]
@sahildua2305 looks like tests are failing?

**Comment by sahildua2305** [comment]
Not sure why `RgbColor` IA  tests are failing. I'll have a look

**Comment by moollaza** [comment]
@sahildua2305 I killed the Travis cache and restarted the test. Everything is good now. I didn't realize the failure was unrelated 👍

**Comment by sahildua2305** [comment]
Thanks @moollaza Looks good now?

**Comment by moollaza** [comment]
@sahildua2305 Thanks! This LGTM :+1:

However it looks like the "Clear" button needs some margin/padding on mobile to keep it from touching the "Validate" button

json_validator_at_duckduckgo

**Pull request merged**